com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/jobs/CleanUpJob.java

```diff
@@ -247,7 +257,11 @@ private void removeProcessLogs(List<String> oldFinishedOperationsIds) {
         return spaceToFileIds;
     }
 
-    private void removeOldFiles(Map<String, List<String>> spaceToFileIds) {
+    private void removeFilesForOperations(List<Operation> operations) {
+        Map<String, String> processIdToAppArchiveId = getProcessIdToAppArchiveId();

```

Comment:
What would you say for such name: getProcessIdToAppArchiveId -> mapProcessIdToArchive?

Reply:
I like it => done